packages/api-clients/open-api/catalogApi.yml

```diff
@@ -1988,6 +1990,39 @@ components:
           type: string
       required:
         - rejectionReason
+    EServiceTemplateInterfaceInfo:
+      type: object
+      additionalProperties: false
+      properties:
+        name:
+          type: string
+        email:
+          type: string
+        url:
+          type: string
+        termsAndConditionsUrl:
+          type: string
+        serverUrl:
+          type: string
+      required:
+        - name
+        - email
+        - url
+        - termsAndConditionsUrl
+        - serverUrl
+    EServiceTemplateInfo:
```

Comment:
usually we don't using "info" postfix in naming convention for components in API definitions of process (exception for BFF).
To avoid ambiguity I suggest a more simply `EServiceTemplate`.